docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,86 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's typically only one @Mod in your mod as a whole, sitting on top of your entry class.
```

Comment:
clarify "chained together" or take it out.

Like said below we assume the modder knows how to specify annotation parameters